core/trino-main/src/main/java/io/trino/operator/BigintGroupByHash.java

```diff
@@ -52,6 +52,8 @@
     private static final float FILL_RATIO = 0.75f;
```

Comment:
TBH I hardly see any difference in microbenchmarks beyond error margin. Not sure it's worth extra complication

Reply:
I guess you are right. I'll drop the commit